src/main/java/hu/eszter/bokkon/model/animal/Animal.java

```diff
@@ -1,5 +1,14 @@
 package hu.eszter.bokkon.model.animal;
 
+import java.util.Map;
+
 public interface Animal {
```

Comment:
Let's think about a way to store the exchange rules at a central space, e.g. here in the Animal interface or in a util class.
So as to avoid storing both in Cow that it is worth 3 Pig and in Pig that it is worth 1/3 Cow.
An NxN matrix maybe? Where N = number of changeable animals. 
In a subsequent pull-request.